appveyor.yml

```diff
@@ -4,18 +4,26 @@ version: '1.0.{build}'
 os: Visual Studio 2015
 
 environment:
+  global:
+    - MAXMIND_BENCHMARK_DB=GeoIP2.Benchmark/GeoLite2-City.mmdb
```

Comment:
Instead of this line, we need: `MAXMIND_TEST_BASE_DIR: MaxMind.MinFraud.UnitTest`